packages/bodiless-demo/src/components/Tables/index.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  addProps, as, on, withDesign, addClassesIf, and, replaceWith, flowHoc,
+} from '@bodiless/fclasses';
+import { withNode, withNodeKey } from '@bodiless/core';
+import { asBodilessChameleon } from '@bodiless/components';
+import {
+  asBodilessTable,
+  useIsInBody,
+  useIsInHead,
+  useIsSecondColumn,
+  useIsThirdColumn,
+  useIsFirstColumn,
+} from '@bodiless/table';
+import { vitalTable, asTableToken } from '@bodiless/vital-table';
+import {
+  EditorPlainClean, RichTextClean, vitalEditorPlain, vitalRichText
+} from '@bodiless/vital-editors';
+import GreenYes from '../../assets/GreenYes';
+import RedNo from '../../assets/RedNo';
+
+const DemoStyleTable = asTableToken({
+  Meta: flowHoc.meta.term('Type')('Table'),
+  Schema: {
+    _: asBodilessTable(),
+    CellContent: as(withNode, withNodeKey('Cell')),
+  },
+  Theme: {
+    Table: 'mx-auto',
+    THead: 'bg-demo-primary-interactive text-white',
+    TFoot: 'bg-demo-primary-interactive',
+    Cell: addClassesIf(and(useIsInBody, useIsFirstColumn))('text-center'),
+  },
+  Spacing: {
+    Cell: 'p-2',
+  },
+  Editors: {
```

Comment:
@wodenx  what did I do wrong with chameleon?  If I haven't swapped the cell, I can swap to these 4 choices, but after first swap, the swap button doesn't come back.

Reply:
probably use `startWith` intead of `replaceWith` -- here `replaceWith` probalby removes the chameleon.